### DISCUSSION TITLE
Vercel doesn't deploy static/~my_folder

### DISCUSSION THREAD

**Discussion by Fitzpasd** [vercel/community#1683] [Help]
### Summary

I have a basic SvelteKit project without any changes to the default configs.  The web app works, but there was no deploy of a folder within my static/ folder.  The dir structure is:

```
- static/
  - ~my_folder/
    - some_third_party_script.js
```

Vercel doesn't deploy `~my_folder` despite it being in the build output.  Is there a config I'm missing?

Confirmed it isn't deployed via a 404 when requesting it from the expected url, and I can see in the 'Source' section on Vercel that it didn't deploy those files.

My expectation is to be able to download those scripts via `http://my_domain.com/~my_folder/some_third_party_script.js`.  This is required for my service worker.

**Comment by amyegan** [comment]
Hi Shane. I wasn't able to replicate the problem. Do you have a public repo or minimal reproduction that I can use to see what's causing the problem for your app? There may be a configuration difference that I'm missing in my test app

**Reply by Fitzpasd** [reply]
Hi Amy, thank you for taking a look.

[Here is a public repo with a minimal repro using the dependencies I am.](https://github.com/Fitzpasd/vercel_repro)

The `static/~partytown` folder is the one that isn't being deployed. [Here is the point](https://github.com/Fitzpasd/vercel_repro/blob/main/vite.config.ts#L9) where vite copies the files I want into `static/~partytown`.  Note that I've added that folder to .gitignore so we don't check those into our repository.

Here you can see that the folder is where I expect it to be when building locally:
![image](https://user-images.githubusercontent.com/1279954/222810998-b44ab89b-d5e6-4531-9712-c679c02398d8.png)

And here's the source output from Vercel (where it doesn't show up):
![image](https://user-images.githubusercontent.com/1279954/222811188-6006e374-545e-4646-9393-1687719fdc49.png)

Finally, I've deployed that to [https://vercel-repro-d1x92mp1o-availabl.vercel.app/](https://vercel-repro-d1x92mp1o-availabl.vercel.app/) and am expecting to have those files available like so: https://vercel-repro-d1x92mp1o-availabl.vercel.app/~partytown/partytown.js

Thanks again for helping, and let me know if there's anything further I can provide to help.

**Reply by Fitzpasd** [reply]
Hi Amy, were you able to reproduce this issue?

**Reply by Fitzpasd** [reply]
Hi Amy, bumping this again.  Is there another channel that I can use to get this issue looked at?  Right now it is blocking our usage of Vercel.
